packages/nimble-components/build/generate-workers/source/index.ts

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+const path = require('path');
+
+const moduleName = './renderWorker.js';
+
+const workersDirectory = path.resolve('./src/wafer-map/workers');
+if (fs.existsSync(workersDirectory)) {
+    console.log(`Deleting existing workers directory "${workersDirectory}"`);
+    fs.rmSync(workersDirectory, { recursive: true });
+    console.log('Finished deleting existing workers directory');
+}
+console.log(`Creating workers directory "${workersDirectory}"`);
+fs.mkdirSync(workersDirectory);
+console.log('Finished creating workers directory');
+
+const modulePath = require.resolve(moduleName);
+const sourceCode = fs.readFileSync(modulePath, 'utf-8');
+
+const componentFileContents =
+`/* eslint-disable no-useless-escape */
+export const workerCode = \`
+${sourceCode.replaceAll('`','\\`').replaceAll('$','\\$').replaceAll('exports.RenderWorker = RenderWorker;','')}
```

Comment:
can you try to see if json parsing the string will make it exportable without the need to replace everything? see https://github.com/ni/nimble/pull/1788#discussion_r1471668308

Reply:
it is working using JSON.stringify